go/chat/bots/commands.go

```diff
@@ -154,46 +154,48 @@ func (b *CachingBotCommandManager) deriveMembersType(ctx context.Context, name s
 	}
 }
 
-func (b *CachingBotCommandManager) createConv(ctx context.Context, param chat1.AdvertiseCommandsParam) (res chat1.ConversationLocal, err error) {
+func (b *CachingBotCommandManager) createConv(
+	ctx context.Context, typ chat1.BotCommandsAdvertisementTyp, teamName *string, convID *chat1.ConversationID,
```

Comment:
i think it's nice to wrap long function signatures but i'm not a fan of newlines between everything here in general